src/full/Agda/Utils/Environment.hs

```diff
@@ -25,6 +28,13 @@ expandVars home env s = concatMap repl $ tokens s
     repl (Var x) = fromMaybe "" $ lookup x env
     repl (Str s) = s
 
+-- | Expand a telescope of environment variables
+--   (each value may refer to variables earlier in the list)
```

Comment:
Please document (somewhere suitable) that the `.flags` file can be a such a telescope (i.e. have such dependencies).

Reply:
Documented in `HACKING.md`